packages/design-system/src/utils/localStore.js

```diff
@@ -37,17 +37,17 @@ function getItemByKey(key) {
   try {
     const stored = localStorage.getItem(key);
     parsed = JSON.parse(stored);
-  } catch (e) {
-    trackError('local_storage_read', e.message);
+  } catch (err) {
+    trackError(err.message);
```

Comment:
```suggestion
    trackError('local_storage_read: ' + err.message);
```
Without the stack trace, the custom name may be helpful to identify the source of errors and categorize them. E.g. multiple `trackError` might have the same error message.

Reply:
Yeah that's a good point. That's the one thing bugging me about exception tracking in GA really.

WDYT about changing the function signature to something like `trackError(prefix, description, fatal = false)` to make that easier? Would make usage a bit more intuitive and require less file changes in this PR.

The `prefix + message` concatenation would then happen inside `trackError`

Reply:
SGTM